org.kde.elisa.json

```diff
@@ -29,6 +30,27 @@
         "/app/lib/vlc/vlc-cache-gen /app/lib/vlc/plugins"
     ],
     "modules": [
+        {
+            "name": "ffmpeg",
+            "config-opts": [
+                "--enable-pic",
+                "--enable-shared",
+                "--disable-doc",
+                "--disable-static",
+                "--enable-gpl",
+                "--enable-libvpx",
+                "--enable-libmp3lame",
+                "--enable-libvorbis",
+                "--enable-libopus"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ffmpeg.org/releases/ffmpeg-4.4.5.tar.xz",
```

Comment:
That's really old and we should likely use the extension instead.

Reply:
I see it's explained in the commit, also: https://github.com/flathub/org.kde.okular/pull/387#issuecomment-2532806441

Reply:
It's old for a reason, the new 6.8 KDE runtime uses the freedesktop 24.08 runtime which has ffmpeg 7. VLC isn't compatible with ffmpeg 7 yet.

The reason I chose 4.4.5 is I looked around and found Solus, Arch and Alpine all keep ffmpeg4 around for compatibility with VLC specifically.

Reply:
Ouch, thanks, makes sense,